src/main/java/uk/gov/companieshouse/model/IllnessReason.java

```diff
@@ -1,16 +1,35 @@
 package uk.gov.companieshouse.model;
 
+import java.util.Collections;
 import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
 
-public class IllnessReason{
+@Valid
+public class IllnessReason extends ReasonType {
 
+    @NotBlank(message = "illPerson must not be blank")
     private final String illPerson;
-    private final String otherPerson;
+
+    @NotBlank(message = "illnessStart must not be blank")
     private final String illnessStart;
+
+    @NotNull(message = "continuedIllness must not be null")
     private final boolean continuedIllness;
-    private final String illnessEnd;
+
+    @NotBlank(message = "illnessImpactFurtherInformation must not be blank")
     private final String illnessImpactFurtherInformation;
-    private final List<Attachment> attachments;
+
+    private final String otherPerson;
+    private final String illnessEnd;
+
+    @Valid
+    private List<Attachment> attachments;
+
+    public IllnessReason(){
+        this(null,null,null,false,null,null,Collections.emptyList());
+    }
```

Comment:
I would advise removing all constructors and using setters to set the field values. It will simplify the code.

Reply:
Resolved on https://github.com/companieshouse/lfp-appeals-api/pull/112